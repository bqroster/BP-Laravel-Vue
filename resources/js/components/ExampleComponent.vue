<template>
<div class="container">
  THIS IS THE PARENT
  <example-nested />
</div>
</template>

<script>
/* eslint import/no-unresolved: 0 */
import ExampleNested from '@component/ExampleNested';

export default {
  components: {
    ExampleNested,
  },
};
</script>
