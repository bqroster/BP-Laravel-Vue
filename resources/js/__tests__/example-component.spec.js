/* eslint no-undef: 0 */
/* eslint import/no-unresolved: 0 */
/* eslint import/extensions: 0 */

import { mount } from '@vue/test-utils';
import ExampleComponent from '../components/ExampleComponent';

describe('ExampleComponent component feature test', () => {
  test('ExampleComponent should be mounted correctly', () => {
    const wrapper = mount(ExampleComponent);
    expect(wrapper.element).toMatchSnapshot();
  });
});
