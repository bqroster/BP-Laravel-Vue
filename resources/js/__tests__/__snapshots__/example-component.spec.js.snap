// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExampleComponent component feature test ExampleComponent should be mounted correctly 1`] = `
<div
  class="container"
>
  
  THIS IS THE PARENT
  
  <div
    class="row justify-content-center"
  >
    <div
      class="col-md-8"
    >
      <div
        class="card"
      >
        <div
          class="card-header"
        >
          Example Component
        </div>
         
        <div
          class="card-body"
        >
          
        I'm an example component.
      
        </div>
      </div>
    </div>
  </div>
</div>
`;
